### PR TITLE
Variable_HW

### PR DESCRIPTION
**Pull request by manita1993**
Review my PR

### CODE REVIEW

File: js_july_23/Users/Manita/HomeWork/Operator_HW/Variables_HW/Variables_HW.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<script>
+    // Given length of a rod in meter, convert the same in centimeter?
+var meter = 5;
+var centimeter;
+
```
Comment (sandeepsainihisar) [inline js_july_23/Users/Manita/HomeWork/Operator_HW/Variables_HW/Variables_HW.html:5]: pls also use camel casing in practise

File: js_july_23/Users/Manita/HomeWork/Operator_HW/Variables_HW/Variables_HW.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+TempInCelcius = (TempInFarenhite - 32)*5/9;
+
+document.write("<br> Temperature in Celcius =" + TempInCelcius);
+document.write("<br>");
+
+
+// Given the principal amount, rate of interest yearly, and time of loan in years, 
+//print the simple interst payable at the end of time duration. 
+
+var Principalamount = 5000;
```
Comment (sandeepsainihisar) [inline js_july_23/Users/Manita/HomeWork/Operator_HW/Variables_HW/Variables_HW.html:24]: camel casing

**Review by sandeepsainihisar — APPROVED** [review]

File: js_july_23/Users/Manita/HomeWork/Operator_HW/Variables_HW/Variables_HW.html (hunk 2 above)
Comment (manita1993) [reply js_july_23/Users/Manita/HomeWork/Operator_HW/Variables_HW/Variables_HW.html:24]: Improved

File: js_july_23/Users/Manita/HomeWork/Operator_HW/Variables_HW/Variables_HW.html (hunk 1 above)
Comment (manita1993) [reply js_july_23/Users/Manita/HomeWork/Operator_HW/Variables_HW/Variables_HW.html:5]: Not an issue

**Pull request merged**